.env.example

```
@@ -18,6 +18,14 @@ VITE_USE_CENTRAL_AUTH=true
 VITE_USE_FALLBACK_AUTH=true
 VITE_PROJECT_SCOPE=dashboard
 
+# AI Service Configuration
+# Set at least one of these for AI-powered features
+ANTHROPIC_API_KEY=sk-ant-xxx  # Preferred - uses Claude for AI responses
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)